src/CppUTest/TestPlugin.cpp

```diff
@@ -62,10 +62,10 @@ void TestPlugin::runAllPostTestAction(UtestShell& test, TestResult& result)
 
 bool TestPlugin::parseAllArguments(int ac, char** av, int index)
 {
-    return parseAllArguments(ac, const_cast<const char**> (av), index);
+    return parseAllArguments(ac, const_cast<const char *const *> (av), index);
```

Comment:
Still have one. But it seems there was already a const_cast here, so I was wrong!